cryptlex/cryptlex-enterprise/values.yaml

```diff
@@ -255,3 +252,10 @@ sso:
   google:
     # Refer to following for enabling Google SSO: https://docs.cryptlex.com/user-management/google-sso
     clientId: ""
+ # Google
+  google:
```

Comment:
please ensure it is a separate section not child of sso, infact we can remove sso section and bring it under google